# exploration_done.srv
bool success 
# Request
---
bool success 


